integration-cli/docker_cli_swarm_test.go

```diff
@@ -1316,3 +1316,18 @@ Resources:
 	c.Assert(err, checker.IsNil, check.Commentf(out))
 	c.Assert(out, checker.Contains, expectedOutput, check.Commentf(out))
 }
+
+func (s *DockerSwarmSuite) TestSwarmUpdateWithoutArgs(c *check.C) {
```

Comment:
@vdemeester That is really great 👍  Integration tests are really costly.

The PR has been updated with `TODO` added.